### PR TITLE
Bc1 33 create a hero section be responsive for app land

### PR DESCRIPTION
**Pull request by MaxRichard2007**

### CODE REVIEW

File: app-land/scss/layout/_heroSection.scss [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+    &--button {
+      width: 9rem;
+
+      .fa-solid {
+        position: relative;
+        font-size: 15px;
+        top: 2px;
+        left: 4px;
+      }
```
Comment (sadiGFG) [inline app-land/scss/layout/_heroSection.scss:47]: Please use rem or em if possible

File: app-land/scss/layout/_heroSection.scss [hunk 2]
```diff
@@ -3,107 +3,102 @@
...
-      p {
-        top: 0rem;
-      }
+  @include retinaMobile {
+    .hero__section--text-overlay {
+      left: 1rem;
+      width: 25rem;
+      font-size: 15px;
     }
   }
```
Comment (sadiGFG) [inline app-land/scss/layout/_heroSection.scss]: Why did you use px again here?

File: app-land/scss/layout/_heroSection.scss [hunk 3]
```diff
@@ -3,107 +3,102 @@
...
+
+  *** FOR MINI MOBILE
+  ** MAX : 400PX / 25em
+  */
+
+  @include mobile {
+    left: 0.5rem;
     .hero__section--text-overlay {
-      max-width: 50%;
+      font-size: 10px;
```
Comment (sadiGFG) [inline app-land/scss/layout/_heroSection.scss:90]: Please fix this too

File: app-land/scss/component/_button.scss [hunk 4]
```diff
@@ -0,0 +1,15 @@
+.cta__button {
+  &--button {
+    width: 7rem;
+    height: 3rem;
+    background: $vibrantGreen;
+    border: none;
+    border-radius: 2rem;
```
Comment (ahmadmohammadirad2006) [inline app-land/scss/component/_button.scss:7]: You can use border radius variables from shared folder when you need border radius in your project  :)

File: app-land/scss/layout/_heroSection.scss [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+      top: 0rem;
+      position: relative;
+      left: 0.5rem;
+      height: 100%;
+
+      h1 {
+        text-align: center;
+        padding-top: 1rem;
+        font-size: 2.2rem;
+        font-weight: 600;
```
Comment (ahmadmohammadirad2006) [inline app-land/scss/layout/_heroSection.scss:37]: Please use font weight variables from shared variables ;-)

File: app-land/scss/layout/_heroSection.scss [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+    .container {
+      width: 95%;
+      top: 0rem;
+      position: relative;
+      left: 0.5rem;
+      height: 100%;
+
+      h1 {
+        text-align: center;
+        padding-top: 1rem;
```
Comment (ahmadmohammadirad2006) [inline app-land/scss/layout/_heroSection.scss:35]: For padding, margin and gap you can use $spaces list from shared variables :)

File: app-land/scss/layout/_heroSection.scss [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+        text-align: center;
+        place-content: center;
+
+        &--button {
+          width: 9rem;
+          border: none;
+          border-radius: 0.5rem;
+          height: 3rem;
+          font-size: 1rem;
```
Comment (ahmadmohammadirad2006) [inline app-land/scss/layout/_heroSection.scss:61]: Again here you can use font size variables from shared folder

**Review by ahmadmohammadirad2006 — CHANGES_REQUESTED** [review]

**Review by sadiGFG — APPROVED** [review]

**Review by ahmadmohammadirad2006 — APPROVED** [review]
Good job bro

**Pull request merged**